src/hooks/NftRewards.ts

```diff
@@ -50,11 +50,12 @@ async function getRewardTierFromIPFS({
 export default function useNftRewards(
   tiers: JB721TierParams[],
   projectId: number | undefined,
+  dataSourceAddress: string | undefined,
 ): UseQueryResult<NftRewardTier[]> {
   const hasTiers = Boolean(tiers?.length)
 
   return useQuery(
-    ['nft-rewards', projectId],
```

Comment:
this is nice to have but doesn't actually fix this specific bug. but we should still have it here.